pkg/proxy/accesslog/record.go

```diff
@@ -65,17 +68,43 @@ const (
 	VersionIPV6
 )
 
-// EndpointInfo contains information about the endpoint sending/receiving the
-// request/response
+// EndpointInfo contains information about the endpoint sending/receiving the flow
 type EndpointInfo struct {
-	ID       uint64
-	IPv4     string
-	IPv6     string
-	Port     uint16
-	Identity uint64
-	Labels   []string
+	ID          uint64
+	IPv4        string
+	IPv6        string
+	Port        uint16
+	Identity    uint64
+	LabelSHA256 string
```

Comment:
I think @manalibhutiyani renamed this to `LabelsSHA256` now.

Reply:
As per Andre's review comments, yes he wanted me to change this to LabelsSHA256. Will be pushing that  out in some time.

Reply:
Done.